testplan/common/utils/logger.py

```diff
@@ -236,9 +236,7 @@ def logger(self) -> TestplanLogger:
         if self._logger:
             return self._logger
 
-        self._logger = logging.getLogger(
-            ".".join([LOGGER_NAME, self.__class__.__name__, uuid4()[-8:]])
-        )
+        self._logger = logging.getLogger(f"{LOGGER_NAME}.{self}")
```

Comment:
e.g

2023-03-29 14:41:31,398  MainProcess  Thread-2     TESTPLAN.GTest[My GTest]       DEBUG           Executing step of GTest[My GTest] - start